packages/wp-babel-makepot/package.json

```diff
@@ -38,7 +38,9 @@
 		"lodash.mergewith": "^4.6.2"
 	},
 	"devDependencies": {
+		"i18n-calypso": "^5.0.0",
```

Comment:
I wonder if we also need to add `@automattic/calypso-build` because it's used in the jest preset this package uses.

Reply:
It's also worth mentioning that the way we're consuming this jest preset (relying on the dir structure vs using it from a package) feels a bit off. Not related to this PR of course

Reply:
Maybe we should move the preset to `@automattic/calypso-build`, I fully agree it is weird to rely on the directory structure.

Reply:
Sounds like a good plan to me.